requirements-integration.in

```diff
@@ -1,4 +1,5 @@
-juju<3.1
+# Pinning to <3.0 to ensure compatibility with the 2.9 controller version
+juju<3.0
 lightkube
 pytest-operator
```

Comment:
Redundant.

Reply:
What do you mean?

Reply:
It's not imported directly and is probably only needed for the asyncio stuff we do on the pytest invocation. You can leave it in I guess.